server/planning/coverage.py

```diff
@@ -16,36 +16,74 @@
 from copy import deepcopy
 from superdesk.errors import SuperdeskApiError
 from superdesk.metadata.utils import generate_guid, item_url
-from superdesk.metadata.item import GUID_NEWSML, metadata_schema
+from superdesk.metadata.item import GUID_NEWSML, metadata_schema, ITEM_STATE
 from superdesk.resource import not_analyzed
 from superdesk import get_resource_service
 from superdesk.notification import push_notification
 from apps.archive.common import set_original_creator, get_user
 from eve.utils import config
-from superdesk.utc import utcnow
-from superdesk.activity import add_activity, ACTIVITY_UPDATE
+from .common import ASSIGNMENT_STATE
+
 
 logger = logging.getLogger(__name__)
 
 
 class CoverageService(superdesk.Service):
     """Service class for the coverage model."""
 
+    def on_fetched_item(self, doc):
+        self.enhance_items([doc])
+
+    def on_fetched(self, docs):
+        self.enhance_items([docs])
```

Comment:
this should be probably `self.enhance_items(docs)`

Reply:
rather it should be `self.enhance_items(docs[config.ITEMS])`